app/models/item.js

```diff
@@ -166,6 +170,26 @@ export default cloudinaryUrl.extend({
     }
   }),
 
+  hasSingleLocation: Ember.computed('packagesLocations.[]', function(){
+    if(this.get('packagesLocations')){
+      return this.get('packagesLocations').length === 1;
+    }
```

Comment:
This can be refactored as `return this.get('packagesLocations') ? this.get('packagesLocations').length === 1 : false`